src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,106 @@
+package ui;
+
+import exceptions.DukeException;
+import task.Task;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import java.util.List;
+
+public class Ui {
+
+    private final BufferedReader br;
+
+    /**
+     * Prints a vertical line.
+     */
+    public static void printVLine() {
+        System.out.println("------------------------------------");
+    }
+
+    /**
+     * Constructor for Ui.
+     */
+    public Ui() {
+        intro();
+        this.br = new BufferedReader(new InputStreamReader(System.in));
+    }
+
+    /**
+     * Reads user input.
+     * @return An array of strings split by spaces.
+     * @throws IOException Throws an error when I/O error is detected.
+     */
+    public String[] getUserCommand() throws IOException {
+        return br.readLine().split(" ");
+    }
+
+    private void intro() {
+        printVLine();
+        System.out.println("Hello! I'm Howie! My hobby is to keep track of TASKS. Let me help you!");
+        System.out.println("So, what can I do for you?");
+        printVLine();
+    }
+
+    /**
+     * Prints the list of tasks that is currently stored by Howie.
+     * @param tasks The collection of tasks.
+     */
+    public static void printAllTask(List<Task> tasks) {
+        printVLine();
+        if (tasks.isEmpty()) {
+            System.out.println("Your list is now EMPTY! Time for you to have a break!");
+            printVLine();
+            return;
+        }
+        System.out.println("You have " + tasks.size() + " left. There are:");
+        for (int i = 1; i <= tasks.size(); i++) {
+            Task t = tasks.get(i-1);
+            System.out.println(i +"." + t);
+        }
+        printVLine();
+    }
+
+    /**
+     * Exits the program when exit command is entered.
+     */
+    public static void exit() {
+        printVLine();
+        System.out.println("Bye! I'll see you when I see you :)");
+        printVLine();
+        System.exit(0);
+    }
+
+    /**
+     * Prints the list of commands available to Howie.
+     */
+    public static void helpMessage() {
+        printVLine();
+        System.out.println("Below is a list of available commands for me:\n" +
```

Comment:
Break before line breaks may be better.  
see: https://se-education.org/guides/conventions/java/intermediate.html#layout